DependencyInjection/Compiler/TwigExceptionPass.php

```diff
@@ -26,5 +26,9 @@ public function process(ContainerBuilder $container)
         if ($container->has('fos_rest.exception_listener') && $container->has('twig.exception_listener')) {
             $container->removeDefinition('twig.exception_listener');
         }
+
+        if (!$container->has('templating.engine.twig')) {
+            $container->removeDefinition('fos_rest.exception.twig_controller');
```

Comment:
Yes, the thing is just if someone injects our service somewhere else (or we start do that at some point for whatever reason), it wouldn't be removed anymore and the error message would be harder to understand imo.